src/main/java/bisq/core/dao/state/blockchain/Tx.java

```diff
@@ -138,14 +138,12 @@ public TxOutput getLastTxOutput() {
 
 
     /**
-     * OpReturn output might contain the lockTime in case of a LockTx. It has to be the last output.
-     * We store technically the lockTime there as is is stored in the OpReturn data but conceptually we want to provide
-     * it from the transaction.
+     * The locktime is stored in the LOCKUP txOutput, which is the first txOutput.
```

Comment:
Hm... I am not sure if the first output is a good place here. Technically its the opReturn output, conceptually its the tx (thats why we use a delegate there).

Reply:
The reason I moved it to the LOCKUP txoutput is that elsewhere it's expected to reside there. It's also a logical place since that's the txoutput that has a restriction on it.